src/lib/chat/index.ts

```diff
@@ -45,6 +46,14 @@ export class Chat {
     this.initSessionHandler(events);
     this.initConnectionHandlers(events);
     this.initChannelHandlers(events);
+
+    // every 5s check if the connection state is CLOSED, if it is, set the app to foreground,
+    // to prevent sendbird sdk from disconnecting (when the app is in the background)
+    setInterval(() => {
```

Comment:
Fine for now but we should move this into the sagas. It should be rare for a component to have to schedule things to happen like this. We could also maybe only do it when the tab is in the background?